zdm/misc_functions.py

```diff
@@ -2017,7 +2023,7 @@ def plot_grid_2(zDMgrid,zvals,dmvals,zmax=1,DMmax=1000,norm=0,log=True,name='tem
 	dz=zvals[1]-zvals[0]
 	if norm==1:
 		zDMgrid /= ddm
-		if Aconst:
+		if Aconts:
```

Comment:
this seems like a typo, i.e. `Aconst` seems correct.

Reply:
The function `plot_grid_2` defines a parameter `Aconts=False`, so I had to change this to` Aconts.` Perhaps the typo is in the function then, and it should be `Aconst=False` ?

Reply:
Esan is corect -  "Aconts" is referring to "contours" for "conts" - i.e. it was likely originally a typo. Perhaps that function was deprecated and I just never used it, it should have failed every time...